python/src/opendp/measurements.py

```diff
@@ -686,7 +687,8 @@ def make_private_lazyframe(
     :param output_measure: How to measure privacy loss.
     :type output_measure: Measure
     :param lazyframe: A description of the computations to be run, in the form of a [`LazyFrame`].
-    :param global_scale: A tune-able parameter that affects the privacy-utility tradeoff.
+    :param global_scale: Optional. A tune-able parameter that affects the privacy-utility tradeoff.
+    :param threshold: Optional. A tune-able parameter that filters smaller partitions.
```

Comment:
I'd assume that the threshold is something like the minimum number of elements in the partition? Wouldn't hurt to be more explicit about what it does?